cosmwasm/packages/wasmi-runtime/src/exports.rs

```diff
@@ -214,11 +214,6 @@ pub unsafe extern "C" fn ecall_query(
     } else {
         *used_gas = gas_limit;
 
-        error!(
-            "ecall_query panic: used_gas {} gas_limit {}",
-            *used_gas, gas_limit
-        );
-
```

Comment:
Reuven found what caused this. `gas_used` in `Enclave.edl` is write-only, so trying to read it caused segfault
https://github.com/enigmampc/SecretNetwork/blob/6537eaead0cdd48bd2a06adc56107c20221107a9/cosmwasm/packages/wasmi-runtime/Enclave.edl#L74